ios/engine/KMEI/KeymanEngine/Classes/InputViewController.swift

```diff
@@ -134,6 +134,13 @@ open class InputViewController: UIInputViewController, KeymanWebViewDelegate {
     super.viewDidAppear(animated)
     setConstraints()
     inputView?.setNeedsUpdateConstraints()
+    
+    //TODO: find out why this is actually happening
+    if(self.containerView?.subviews.count == 0)
```

Comment:
`containerView` can be referenced without `self` and `isEmpty` can be used in place of `count == 0`.